docs/tutorials/timeseries/advanced/forecasting-custom-model.ipynb

```diff
@@ -134,26 +134,13 @@
     "        # to define a custom preprocessing logic\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #83.                default_hyperparameters["devices"] = 1
</pre><p>It seems that a return statement is missing here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/autogluon/autogluon/pull/5038/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
thank you!